xcm/xcm-executor/src/traits/mod.rs

```diff
@@ -44,14 +42,17 @@ pub use should_execute::{CheckSuspension, Properties, ShouldExecute};
 mod transact_asset;
 pub use transact_asset::TransactAsset;
 mod weight;
+#[deprecated = "Use `sp_runtime::traits::` instead"]
+pub use sp_runtime::traits::{Identity, TryConvertInto as JustTry};
 pub use weight::{WeightBounds, WeightTrader};
 
 pub mod prelude {
 	pub use super::{
-		export_xcm, validate_export, AssetExchange, AssetLock, ClaimAssets, Convert, ConvertOrigin,
-		Decoded, DropAssets, Enact, Encoded, Error, ExportXcm, FeeManager, FeeReason, Identity,
-		JustTry, LockError, MatchesFungible, MatchesFungibles, MatchesNonFungible,
-		MatchesNonFungibles, OnResponse, ShouldExecute, TransactAsset, VersionChangeNotifier,
-		WeightBounds, WeightTrader, WithOriginFilter,
+		export_xcm, validate_export, AssetExchange, AssetLock, ClaimAssets, ConvertOrigin,
+		DropAssets, Enact, Error, ExportXcm, FeeManager, FeeReason, LockError, MatchesFungible,
+		MatchesFungibles, MatchesNonFungible, MatchesNonFungibles, OnResponse, ShouldExecute,
+		TransactAsset, VersionChangeNotifier, WeightBounds, WeightTrader, WithOriginFilter,
 	};
+	#[allow(deprecated)]
+	pub use super::{Identity, JustTry};
```

Comment:
Are we going to continue exporting these types after deprecation by re-exporting from sp_runtime? What's the plan otherwise?

Reply:
We'll remove them eventually (hence deeprecation), but they're here to minimise breaking changes.